package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "tinypg",
-  "version": "7.0.1",
+  "version": "8.0.0",
   "description": "Easy way to call sql files using postgres.",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "scripts": {
-    "build-test": "npm run clean && tsc -p src/test",
```

Comment:
intentionally delete the clean step?

Reply:
I can put it back. I was trying to figure out why the CI was failing. Turns out it has to do with TS and lodash types.